### ISSUE TITLE
Bad performence on pen environment

### ISSUE THREAD

**Issue by cccedric** [Zhendong-Wang/Diffusion-Policies-for-Offline-RL#8]
Hi Zhengdong,

I use the DQL code and the hyperparameters the code provided to test the algorithm on pen-cloned-v1 and pen-human-v1, but the results I got is far away from what the paper said. The average score can only reach about 28. And the critic loss will be exploded to a crazy value (about 1e10). 

The evaluation result is shown below:
![Screenshot from 2023-06-22 11-30-49](https://github.com/Zhendong-Wang/Diffusion-Policies-for-Offline-RL/assets/65392464/24e7ece2-8c8f-47c6-8160-dfc798bdb074)

The target Q mean result is shown below:
![Screenshot from 2023-06-22 11-31-30](https://github.com/Zhendong-Wang/Diffusion-Policies-for-Offline-RL/assets/65392464/afd6d509-c102-4bfa-bdf9-70beffcf67e4)

The critic loss result is shown below:
![Screenshot from 2023-06-22 11-32-23](https://github.com/Zhendong-Wang/Diffusion-Policies-for-Offline-RL/assets/65392464/fe67740f-1724-4aad-8d21-4537622cfbe2)

Have you met the same issue and how to solve it? 

Thanks!

**Comment by Zhendong-Wang** [comment]
I am not sure which model selection method you are using. If you are using offline, the model training should stop before the critic values go crazy. For online setting, I remembered that I met critic value exploding sometimes for Adroit tasks, but it won't affect the highest performance, where the final model is selected. Or you could choose to strenghten the policy regularization part to avoid the critic exploding. 

I rerun some experiments, and on my machine the performance matched.

**Comment by HenryZhang-git** [comment]
Hi,bro.Could you tell me how to visualize data?I have trained agent and had a file which named debug.log.Your reply is so important to me.Thank you very much!

**Comment by cccedric** [comment]
I use tenserboard to visualize the data :)

**Comment by HenryZhang-git** [comment]
> I use tenserboard to visualize the data :)

Thanks for your reply!I also want to know, did you run this project on Ubuntu and use Tensorboard for visualization？Thank you very much!

**Comment by cccedric** [comment]
> 

Yes, I run the project on ubuntu20.04.

**Comment by HenryZhang-git** [comment]
> > 
> 
> Yes, I run the project on ubuntu20.04.

Your reply is very helpful to me！Thank you!Looking forward to our next communication！
